cluster/examples/kubernetes/minio/object-store.yaml

```diff
@@ -23,6 +23,14 @@ metadata:
 spec:
   scope:
     nodeCount: 4
+    volumeClaimTemplates:
+    - spec:
+        accessModes: [ "ReadWriteOnce" ]
+        # Set the storage class that will be used, otherwise Kubernetes' default storage class will be used.
```

Comment:
the comment used in the cockroachdb/cluster.yaml is different than this one, want to make them consistent?